skill-tree /src/main/java/com/RDS/skilltree/Endorsement/EndorsementController.java

```diff
@@ -0,0 +1,44 @@
+package com.RDS.skilltree.Endorsement;
+
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.util.Map;
+import java.util.UUID;
+
+import static com.RDS.skilltree.Endorsement.JsonApiResponseConverter.convertToHashMap;
+
+@RestController
+@RequestMapping("/v1/endorsements")
+@Slf4j
```

Comment:
why have we used this `Slf4j` here ?

Reply:
it is used for the purpose of logging and is not mandatory. It was implemented in the Skills controller, so I thought to add it here as well to stay consistent.